cmd/executor/cmd/root.go

```diff
@@ -84,10 +84,13 @@ var RootCmd = &cobra.Command{
 			logrus.Error(err)
 			os.Exit(1)
 		}
-		if err := executor.DoPush(ref, image, destination, tarPath); err != nil {
-			logrus.Error(err)
-			os.Exit(1)
+
+		for _, destination := range destinations {
```

Comment:
Could we instead pass the destinations into DoPush, and have the for loop there?